app/views/users/_sidebar.html.erb

```diff
@@ -5,7 +5,7 @@
   <li><%= span_if_current ts("Profile"), user_profile_path(@user) %></li>
   <% if @user.pseuds.size > 1 %>
     <li class="pseud" aria-haspopup="true">
-      <% pseud_link_text = current_page?(@user) ? ts("Pseuds") : (@author ? @author.name : @user.login) %>
+      <% pseud_link_text = ts("Pseuds") %>
```

Comment:
Can we do `<a href="#" title="<%= ts("Pseud Switcher") %>"><%= ts("Pseuds") %></a>` instead? I don't think there's much value to continuing to use the `pseud_link_text` variable, since it looks like it's only used in that one place.

Reply:
Done! ☺️ It's been a bit since I've worked with Ruby, so I'll just ascribe that to growing pains. Thanks for the kind review!